nilearn/masking.py

```diff
@@ -82,8 +82,9 @@ def _extrapolate_out_mask(data, mask, iterations=1):
     outer_shell[1:-1, 1:-1, 1:-1] = np.logical_xor(new_mask, mask)
     outer_shell_x, outer_shell_y, outer_shell_z = np.where(outer_shell)
     extrapolation = list()
-    for i, j, k in [(0, 1, 0), (0, -1, 0), (1, 0, 0), (-1, 0, 0),
-                    (1, 0, 0), (-1, 0, 0)]:
+    for i, j, k in [(1, 0, 0), (-1, 0, 0), 
+                    (0, 1, 0), (0, -1, 0),
+                    (0, 0, 1), (0, 0, -1)]:
```

Comment:
```
an_appropriate_name = [(1, 0, 0), (-1, 0, 0), (0, 1, 0),   # Arranged the way it will be accessed in the for loop.
                       (0, -1, 0), (0, 0, 1), (0, 0, -1),
                       ]
for i, j, k in an_appropriate_name:
```
To make this code easier to comprehend down the road, when we may revisit it, i suggest appropriate names for the variables `i`,`j`, `k`, and `an_appropriate_name`.

Reply:
Agreed for `an-appropriate_name`, but not so for `i, j, k` which are natural array indices ?

Reply:
It wasn't clear to me they are array indices when I only looked at this short code sample from the PR. I will leave this to your discretion.